entc/gen/template/internal.tmpl

```diff
@@ -17,3 +17,14 @@ package internal
 
 const Schema = {{ .SchemaSnapshot }}
 {{ end }}
+
+{{ define "internal/globalid" }}
+
+{{ with $.Header }}{{ . }}{{ else }}// Code generated by ent, DO NOT EDIT.{{ end }}
+
+// +build tools
+
+package internal
+
+const IncrementStarts = {{ .Annotations.IncrementStarts | json | quote }}
```

Comment:
Maybe add a GoDoc here,

Reply:
Followed the rest of the template (where there is no GoDoc).